build.xml

```diff
@@ -38,89 +57,258 @@ where:
   <condition property="saxon.jar" value="${saxon.jar.unix}">
     <os family="unix" />
   </condition>
-
+  
   <property name="saxon.home" location="/Library/Application Support/saxonb9" />
   <property name="saxon.jar" location="${saxon.home}/saxon9.jar" />
-
+  
   <dirname property="xspec.base.dir" file="${xspec.xml}" />
   <basename property="xspec.xml.base" file="${xspec.xml}" suffix=".xml" />
   <basename property="xspec.base" file="${xspec.xml.base}" suffix=".xspec" />
-
+  
   <property name="xspec.dir"
     value="${xspec.base.dir}/xspec" />
-
+  
   <property name="xspec.xsl"
     value="${xspec.dir}/${xspec.base}.xsl" />
   <property name="xspec.result.xml"
     value="${xspec.dir}/${xspec.base}-result.xml" />
   <property name="xspec.result.html"
     value="${xspec.dir}/${xspec.base}-result.html" />
   
-	<macrodef name="xspec"
-						description="Run XSpec tests">
-		<attribute name="xspec.xml" description="XSpec tests file" />
-		<sequential>
-			<local name="xspec.base.dir" />
-			<dirname property="xspec.base.dir" file="@{xspec.xml}" />
-			<local name="xspec.xml.base" />
-			<basename property="xspec.xml.base" file="@{xspec.xml}" suffix=".xml" />
-			<local name="xspec.base" />
-			<basename property="xspec.base" file="${xspec.xml.base}" suffix=".xspec" />
-			<antcall target="xspec.xspec" inheritall="false">
-				<param name="xspec.xml"
-							 location="@{xspec.xml}" />
-				<param name="xspec.base.dir"
-							 location="${xspec.base.dir}" />
-				<param name="xspec.dir"
-							 location="${xspec.base.dir}/xspec" />
-				<param name="xspec.base"
-							 value="${xspec.base}" />
-			</antcall>
-		</sequential>
-	</macrodef>
+  <!-- OXYGEN PATCH needed to create a custom Saxon, for example  -warnings:recover -strip:none -opt:10 -dtd:off -l:off -versionmsg:off -expand:on -outval:fatal -val:lax -->
```

Comment:
Once the code is merged into `master` I suggest to remove `OXYGEN PATCH` and `OUP PATCH` but keep all the relevant comments.